src/dlgprefvinyldlg.ui

```diff
@@ -734,7 +734,7 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
-         <string>&lt;a href=&quot;http://www.mixxx.org/wiki/doku.php/vinyl_control#troubleshooting&quot;&gt;Troubleshooting&lt;/a&gt;</string>
+         <string>&lt;a href=&quot;http://mixxx.org/manual/latest/chapters/vinyl_control.html#troubleshooting&lt;/a&gt;</string>
```

Comment:
Someone in the Qt IRC channel mentioned that the variables would have to be moved to a .cpp file.